playbooks/gce/openshift-node/config.yml

```diff
@@ -1,49 +1,123 @@
----
-- name: "populate oo_hosts_to_config host group if needed"
+- name: node/config.yml, populate oo_nodes_to_config host group if needed
   hosts: localhost
   gather_facts: no
   tasks:
   - name: Evaluate oo_host_group_exp
-    add_host: "name={{ item }} groups=oo_hosts_to_config"
+    add_host: "name={{ item }} groups=oo_nodes_to_config"
     with_items: "{{ oo_host_group_exp | default('') }}"
     when: oo_host_group_exp is defined
+  - name: Find masters for env
+    add_host: "name={{ item }} groups=oo_masters_for_node_config"
+    with_items: groups['tag_env-host-type-' + oo_env + '-openshift-master']
```

Comment:
Good catch, it is not. See previous note about variable precedence and oo_nodes_to_config. If I can sort out the variable precedence issues, then I'll go back to using oo_masters_for_node_config, since it cleans up readability for later references to master hosts.